code/modules/projectiles/projectile.dm

```diff
@@ -350,6 +353,9 @@
 		if(PROJECTILE_FROZEN)
 			return PROCESS_KILL
 
+	if(QDELETED(src))
+		return PROCESS_KILL
```

Comment:
how would it be qdelted here, its checked above

Reply:
ditto, in batch move